project/Vagrantfile

```diff
@@ -91,11 +102,73 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
                 data.vm.network "public_network", bridge: 'en0: Wi-Fi (AirPort)'
                 data.vm.synced_folder "/opt/vagrant", "/opt/vagrant"
                 data.vm.synced_folder "/space/software", "/space/software"
-                
+
                 data.vm.provision :hostmanager
                 data.vm.provision :shell, :path => "/opt/vagrant/bootstrap-centos-extra.sh", :args => ["-v", "#{@ml_version}", mastername, hostname, File.basename(Dir.getwd)]
             end
         end
     end
-    
+
+end
+module UpdateCentOSKernel
```

Comment:
Could be. To my understanding this is an issue with CentOS 6.5, could be that another vm or using a newer version of CentOS could solve this.